00 Intro/00 BoilerPlate/README.md

```diff
@@ -16,7 +16,7 @@ Summary steps:
 
 ## Prerequisites
 
-Prerequisites, you will need to have nodejs installed in your computer.
+Prerequisites, you will need to have [nodejs](https://nodejs.org/en/) (v. 6.3.1) installed in your computer.
```

Comment:
It would be a good idea to add the version in the link itself something like:

nodejs (v.6.3.1)